core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -615,11 +617,13 @@ oppia.directive('conversationSkin', [
             $scope.startCardChangeAnimation = true;
 
             $timeout(function() {
-              var newInteractionHtml = oppiaPlayerService.getInteractionHtml(
+              var newInteractionHtml =
+              ExplorationPlayerService.getInteractionHtml(
```

Comment:
Indent it once from `var` like 
![indent2](https://user-images.githubusercontent.com/7807557/31108069-68219950-a814-11e7-83db-27429f5d0a22.jpeg)